index.html

```diff
@@ -120,24 +121,31 @@ <h3>Postdocs</h3> <!-- Ordered by last name, Research Scientists and Post Doctor
 
 <h3>Software Engineers</h3> <!-- Ordered by last name, Software Engineers in IC speak -->
 <ul>
-<li>Antony Vennard</li>
+<li>Pierluca Borsò (DEDIS)</li>
+<li>Noémien Kocher (DEDIS)</li>
+<li>Antony Vennard</li>    
+<li>Jean Viaene (DEDIS)</li>
```

Comment:
Maybe the other people should have the associated lab next to their names. Since there are no links on this page to the lab pages, how do people know who is connected to what lab?